### ISSUE TITLE
m3u playlists not being auto imported due to "no admin users"

### ISSUE THREAD

**Issue by deafmute1** [navidrome/navidrome#1546]
### Description

Navidrome fails to import m3u playlist files due to "Playlists will not be imported, as there are no admin users yet, Please create an admin user first, and then update the playlists for them to be imported"

### Expected Behaviour

m3u files should be auto imported as playlists.

### Steps to reproduce

1. Add m3u to library directory
2. They are not auto imported during scan

### Platform information

 - Navidrome version: 0.47.0 (8c7d95c1)
 - Browser and version: N/A - backend issue
 - Operating System: openmediavault 5.6.20-1/Debian 10; docker container

### Additional information

Example of a log produced on failure to import playlist:

```21-12-31T12:14:13Z" level=warning msg="Playlists will not be imported, as there are no admin users yet, Please create an admin user first, and then update the playlists for them to be imported" dir="/music/the pillows/the pillows - 2008 - PIED PIPER [FLAC]"```

This same log is produced for every `.m3u` file in the library.

For further context, there is of course an admin user already:
![image](https://user-images.githubusercontent.com/32427188/147824251-f341f997-eb81-4f11-8bfb-6c66e901a13f.png)

**Comment by deafmute1** [comment]
Okay, I took the warning messages advice and "updated the playlists" by adding an underscore to the end of their name (`playlist.m3u` -> `playlist_.m3u`). This allowed the playlists to be imported. 

A simple batch rename workaround can assist you in this situation: `rename 's/\.m3u$/_.m3u/' *` 

However, this is still unexpected behavior and a bug, so I will leave this issue open.

**Comment by metalheim** [comment]
Yeah. to avoid that, navidrome should either: 
only start scanning after the first admin user was created 
Do a full Re-Scan after first admin user was created
Re-parse Playlists (and assign to admin user) after first user was created

**Comment by tokenwizard** [comment]
I'm having issues here with Playlists, too. I had an instance running on one PC and created a nice Playlist of some of my favorite songs. I just installed a new Instance of Navidrome on my new Homelab server and copied my m3u playlist over from the old instance and it does not show up in my Library. I went as far as to create a new playlist in the new instance, then searched the entire hard drive of m3u and m3u8 files, and the new playlist is not found. I also searched for any files that are named to match my new playlist. So where/how are playlists stored if they are not in files of some sort? Are they contained entirely in the Database now?

I have an "admin" user as well as another account that is flagged as an admin. I tried moving the m3u playlist to another folder and then back into the music library as well as "touch"-ing the playlist file and it still does not show; only the new playlist I created trying to test shows in the Playlists section, and THAT playlist cannot be found in any file on the server.
